tests/main/interfaces-calendar-service/task.yaml

```diff
@@ -21,7 +22,14 @@ prepare: |
 restore: |
     kill "$(cat dbus-launch.pid)"
     rm -f dbus-launch.pid
-    rm -rf "$XDG_CONFIG_HOME" "$XDG_DATA_HOME" "$XDG_CACHE_HOME"
+
+    # In case the process gvfsd-metadata does not finish by itself, it is manually stopped
+    # The reason is that gvfsd-metadata locks the xdg/share/gvfs-metadata directory content
+    # producing an error when the xdg directory is removed.
+    if pid="$(pidof gvfsd-metadata)"; then
+        kill -9 "$pid" || true
```

Comment:
-9 is a bit harsh, can we ask it nicely? Can we kill the right version instead of any random one (likely >1 on non-test system).

Reply:
It's not a negotiation with gvfsd :smile: I'm hoping that since we're running in a controller environment, there will be just one instance.